solutions/secure-cross-regional-bucket/variables.tf

```diff
@@ -0,0 +1,189 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Token"
+  sensitive   = true
+}
+
+########################################################################################################################
+# KMS variables
+########################################################################################################################
+
+variable "kms_region" {
+  type        = string
+  default     = "us-south"
+  description = "The region in which KMS instance exists."
+}
+
+variable "existing_kms_guid" {
+  type        = string
+  default     = null
+  description = "The GUID of of the KMS instance used for the SCC COS bucket root Key. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
```

Comment:
Need to update var description according to this DA.
- `of of` -> of
- SCC? I think we copied from there.
- are we using `skip_cos_kms_auth_policy` this variable? I think no.